ReactiveCocoa/Swift/Flatten.swift

```diff
@@ -627,12 +629,13 @@ extension SignalType where Value: SignalProducerType, Error == Value.Error {
 		}
 	}
 
-	private func observeSwitchToLatest(observer: Observer<Value.Value, Error>, _ latestInnerDisposable: SerialDisposable) -> Disposable? {
+	@discardableResult
```

Comment:
I think we shouldn't ignore this being discarded, since it returns a `Disposable`.